hack/update-all.sh

```diff
@@ -0,0 +1,34 @@
+#!/bin/bash
+
+# Copyright 2014 The Kubernetes Authors All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+set -o errexit
+set -o nounset
+set -o pipefail
+
+KUBE_ROOT=$(dirname "${BASH_SOURCE}")/..
+source "${KUBE_ROOT}/cluster/kube-env.sh"
+
+BASH_TARGETS="generated-conversions
+	generated-deep-copies 
+	generated-docs 
+	generated-swagger-docs 
+	swagger-spec"
+
+for t in $BASH_TARGETS
+do
+	echo -e "Updating $t"
+	bash "$KUBE_ROOT/hack/update-$t.sh" 1> /dev/null || echo -e "${color_red}FAILED${color_norm}" 
```

Comment:
Sending stdout to /dev/null makes debugging difficult. Perhaps better to let the caller decide where to send it?